forge/fileServer.js

```diff
@@ -50,6 +50,11 @@ module.exports = async (options = {}) => {
         logger: loggerConfig
     })
 
+    if (runtimeConfig.telemetry.backend?.prometheus?.enabled) {
```

Comment:
```suggestion
    if (runtimeConfig.telemetry?.backend?.prometheus?.enabled) {
```

AFAICT we cannot assume `telemetry` exists either?

Reply:
yep, makes sense